internal/cri/server/sandbox_stats_list.go

```diff
@@ -34,20 +35,32 @@ func (c *criService) ListPodSandboxStats(
 	r *runtime.ListPodSandboxStatsRequest,
 ) (*runtime.ListPodSandboxStatsResponse, error) {
 	sandboxes := c.sandboxesForListPodSandboxStatsRequest(r)
+	stats, errs := make([]*runtime.PodSandboxStats, len(sandboxes)), make([]error, len(sandboxes))
+
+	eg := new(errgroup.Group)
+	for i, sandbox := range sandboxes {
+		i := i
+		eg.Go(func() error {
```

Comment:
It looks like there are no errors ever returned from a goroutine, so this could be a `sync.WaitGroup` instead.  Or we could use it as an `errgroup.Group` and remove the manual `errs` slice management.

Reply:
SG! I previous thought `errgroup.Group` was cleaner for code but any error returned in one of the goroutines would stop all remaining goroutines, which is not wanted.